ui/pages/confirmations/components/confirm/info/shared/transaction-details/transaction-details.tsx

```diff
@@ -28,12 +30,14 @@ const OriginRow = () => {
   }
 
   return (
-    <ConfirmInfoRow
+    <ConfirmInfoAlertRow
+      alertKey="originSpecialCharacterWarning"
```

Comment:
If the issue is applying the same alert to multiple different fields, then would a simpler solution be to just return two alert instances using the same base object, but with alternate field properties.

Though for this scenario, we shouldn't have simultaneous signature and transaction confirmations, meaning could we also just have the hook return a single alert with a dynamic `fieldKey` based on the confirmation type?